ApplicationInsightsInternalLogger/src/main/java/com/microsoft/applicationinsights/internal/logger/InternalLogger.java

```diff
@@ -256,7 +321,7 @@ private void setLoggerOutput(String loggerOutputType, Map<String, String> logger
                 // If the user asked for a logger type
                 type = LoggerOutputType.valueOf(loggerOutputType.toUpperCase());
             } catch (Exception e) {
-                System.out.println(e);
+                System.err.println(e);
```

Comment:
This should be output to stderr and though I could make a case for it being removed.